### PR TITLE
list: add unindent all function

### PR DESCRIPTION
**Pull request by mneira10**
## Proposed Changes

This adds a function to unindent the list item to the base of the list, regardless of the indentation level its in. 

I wrote this up after needing it in my personal projects. WDYT?

### CODE REVIEW

File: list/list.go [hunk 1]
```diff
@@ -120,6 +120,12 @@ func (l *List) UnIndent() {
 	}
 }
 
+func (l *List) UnIndentAll() {
+	for l.level > 0 {
+		l.level--
+	}
+}
```
Comment (jedib0t) [inline list/list.go]: This entire for loop can be reduced to:
```golang
l.level = 0
```

**Review by jedib0t — CHANGES_REQUESTED** [review]
Can you please add a unit-test to verify this functionality and ensure that Render() renders properly and does not get broken?

Edit: I really appreciate the contribution! ❤️

File: list/list.go (hunk 1 above)
Comment (mneira10) [reply list/list.go]: Right of course. Why didn't I see that 😳

**Comment by mneira10** [comment]
Ok, added some tests!

**Review by jedib0t — APPROVED** [review]
Thanks for the contribution!

File: list/render_test.go [hunk 2]
```diff
@@ -289,3 +289,47 @@ func TestList_Render_Styles(t *testing.T) {
...
+    * This
+    * Is
+    * Known
+* The Dark Tower
+  * The Gunslinger
+* The Mandalorian
+  * This
+    * Is
+      * The
+        * Way
```
Comment (jedib0t) [inline list/render_test.go:331]: ❤️

**Comment by jedib0t** [comment]
@mneira10 I've merged the PR. Let me know if you need a tag, or if it can wait.

**Comment by mneira10** [comment]
A tag would be nice! But I'm not in a hurry, I can wait for it to get bundled with other stuff for a little bit

**Comment by jedib0t** [comment]
Tags are cheap. I’ll get you one within the hour.
________________________________

**Comment by jedib0t** [comment]
Here you go: https://github.com/jedib0t/go-pretty/releases/tag/v6.3.6

**Pull request merged**